<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="Desktop-22" failures="1" tests="2" name="Dashboard.Customer_Landing.TC_20_TC_30_Verify_Report_View_Functionality" time="208.748" errors="0" timestamp="2025-07-09T12:53:39 IST" skipped="0">
  <testcase classname="Dashboard.Customer_Landing.TC_20_TC_30_Verify_Report_View_Functionality" name="TC_30_Verify_pictorial_scale_in_detailed_report" time="0.605">
    <failure message="Missing test in UI: GON" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: Missing test in UI: GON
at org.testng.Assert.fail(Assert.java:110)
at Test.Dashboard.Actions_Class.Customer_Landing_Actions.verifyBiomarkerDetailsSHTest(Customer_Landing_Actions.java:1070)
at Dashboard.Customer_Landing.TC_20_TC_30_Verify_Report_View_Functionality.TC_30_Verify_pictorial_scale_in_detailed_report(TC_20_TC_30_Verify_Report_View_Functionality.java:23)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:136)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:658)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:219)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:923)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:192)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
at org.testng.TestRunner.privateRun(TestRunner.java:808)
at org.testng.TestRunner.run(TestRunner.java:603)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:429)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:423)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:383)
at org.testng.SuiteRunner.run(SuiteRunner.java:326)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1249)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
at org.testng.TestNG.runSuites(TestNG.java:1092)
at org.testng.TestNG.run(TestNG.java:1060)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
    </failure>
  </testcase> <!-- TC_30_Verify_pictorial_scale_in_detailed_report -->
  <system-out/>
  <testcase classname="Dashboard.Customer_Landing.TC_20_TC_30_Verify_Report_View_Functionality" name="TC_20_Verify_report_view_functionality" time="208.143"/>
  <system-out/>
</testsuite> <!-- Dashboard.Customer_Landing.TC_20_TC_30_Verify_Report_View_Functionality -->
